DataDog/documentation
# Move more build scripts

### What does this PR do? What is the motivation?

- Updates the make commands to account for more scripts being removed/moved
- Updates the gitignore
- Updates the CI to call a script that handles the moving/sourcing of the build files that have moved 
### Merge instructions



Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### Additional notes

Testing:
1. Pull the branch down
2. Reach out to me for the make config vars
3. Run `make start`
4. ensure `bin/py` directory has all the files you'd expect to see
5. ensure the `bin/js` dir and file shows up
6. ensure the `bin/py/translations` dir and files show up
7. https://gitlab.ddbuild.io/DataDog/documentation/-/jobs/727416884#L96 <- this is a build pulling from the image with the most up to date moves. This output was for debugging to ensure the copy script is doing what we expect, it has been removed from the script.
